ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master.yaml

```diff
@@ -88,6 +88,7 @@ tests:
 - as: ocs-operator-e2e-aws
   commands: |
     make functest
+    make shellcheck-test
```

Comment:
Just add `shellcheck-test` as a dependency of the `ocs-operator-ci` target in the ocs-operator Makefile. This is not needed.

Reply:
Huh, I see that it's already a dependency on the ocs-operator-ci target. This PR is not actually needed. I'm closing it.

/close